### DISCUSSION TITLE
Embeddings

### DISCUSSION THREAD

**Discussion by zmichels** [mtex-toolbox/mtex#658] [Ask Anything]
Hello,

The latest versions of MTEX (I am using 5.6.beta.1) includes support for defining "an embedding of the orientation space into an Eucledian space." At the bottom of the documentation page about embeddings (https://mtex-toolbox.github.io/OrientationEmbeddings.html)... it says...

Operations
The following operations are supported for embedding:
```
+, -, *, .*, ./
sum, mean
norm, normalize
dot
rotate, rotate_outer
```

However, for two embeddings defined from orientations, e1 and e2, if I try the operations `*`, `.*`, or `./`, I get errors, such that these operations do not [yet] seem supported. I have also tried `mtimes` and other functions/operations, to no avail. Can anyone help me figure out how to use the `*` or `.*` operations on these? 

Sidenotes:
I can apply those operations after converting to double (i.e., double(e1)), but then I have the trouble that:
1. I am not totally sure it is doing what I want, and
2. I am unsure how to get back to an embedding or orientation representation of the result (now a double).

Thanks for any help.

Cheers,
Zach

edit: for some reason, the comment window I am using does not publish the asterisk signs for the operations. I want to "multiply" them

**Comment by ralfHielscher** [comment]
Hi Zach,

until now the operation `.*` was indeed only implemented for products between a scalar and an embedding. In 28f3cbf I added support for `embedding .* embedding` as well as `embedding ./ embedding` . Would you like testing this?

I'm not so sure how to interpret `embedding * embedding` .

All the best,

Ralf.

**Comment by zmichels** [comment]
Hi Ralf,

That’s great, I’ll check out that version.

As far as the * operator, I guess I don’t know what I was expecting really either. I think I was just imagining using it similar to a rotation*rotation, but  .* and ./ are great. 

Thank you!

Zach
